A <%= @exception.class %> occurred in <%= @controller.controller_name %>#<%= @controller.action_name %>:

  <%= @exception.message %>
  <%= @backtrace.first %>
<% unless @the_blamed[:author].nil? %>
Git Blame: '<%=@the_blamed[:author]%>' last modified line #<%=@the_blamed[:line]%> in <%=@the_blamed[:file]%>
<% end%>
<%= @sections.map { |section| render_section(section) }.join %>
